libpod/in_memory_state.go

```diff
@@ -315,9 +436,18 @@ func (s *InMemoryState) HasPod(id string) (bool, error) {
 		return false, ErrEmptyID
 	}
 
-	_, ok := s.pods[id]
+	pod, ok := s.pods[id]
+	if ok {
```

Comment:
Check if !ok
return ok, nil
THen the following lines can be combined into a simple return code.